src/Microsoft.VisualStudio.Extensibility.Testing.Xunit.IntegrationTests.Shared/VisualStudioInstanceKeyTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for more information.
+
+namespace Microsoft.VisualStudio.Extensibility.Testing.Xunit.IntegrationTests
+{
+    using System;
+    using global::Xunit;
+    using global::Xunit.Harness;
+
+    public class VisualStudioInstanceKeyTests
```

Comment:
It would be interesting to see if an env var with `=` in its value would be properly carried all the way over to the `ProcessStartupInfo` block somehow.

Reply:
➡️ Added integration tests that set custom environment variables